multi-node-test.sh

```diff
@@ -235,8 +195,13 @@ else
     rc=0
 fi
 
-scp -i ci_key -r jenkins@"${nodes[0]}:\
-$CART_BASE/install/Linux/TESTING/$log_base_path" install/Linux/TESTING/
+ssh -i ci_key jenkins@"${nodes[0]}" "set -x
+ls -l /var/tmp
+find $log_base_path -type f"
+
+scp -i ci_key -r jenkins@"${nodes[0]}":\
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]